README.md

```diff
@@ -296,6 +296,13 @@ TBD:
 	hash_index_cells_used
 	additional_pool_alloc
 
+### InnoDB metrics from information_schema
+
+Collected by parsing the output of `SELECT COUNT from INFORMATION_SCHEMA.INNODB_METRICS where name ='os_log_bytes_written';`:
+
+    mysql plugin: Sending value: innodb/os_logs_bytes_written=263167952896
```

Comment:
why not being consistent with the name of source metric : os_log_bytes_written
therefore innodb/os_log_bytes_written

Reply:
OK looks good to me.